bridge/README.md

```diff
@@ -133,6 +133,32 @@ The config file itself does the heavy lifting.
 
 When unset, the current working directory is checked for a file named `svix-bridge.yaml`.
 
+## Variable Expansion
+
+`svix-bridge` supports variable expansion inside the config file, when launched with `--expand-vars` or
```

Comment:
Why make it a configuration value? Just have it always on, I can't think when we would even want it off.

Reply:
Can do.

There was an earlier concern about security, but since this implementation doesn't actually evaluate expressions of any kind, the worst thing that can happen is leaking your env vars to yourself (not a real problem).

An edge case: things that look like vars but aren't meant to be replaced. With the current implementation, the original token is retained (which would be good) unless there's actually a matching env var set.

Escaping would solve for this case, but we don't have that currently.

Reply:
I think there's no security concern and this is just a source for confusion (why isn't it working?!).